### ISSUE TITLE
Tables/Plots and Loading Time

### ISSUE THREAD

**Issue by agropunk** [rhull21/re_app#61]
One issue I'm noticing is that plots take awhile to load for me, even when they should be cached. Because of this, in pages that contain both a plot and table, opening up new pages in the table takes awhile because the plot needs to reload each time. e.g. the Discharge time Series page

I'm not sure if there's a way to just load the table part of the page, but otherwise I'd suggest having tables and plots as separate pages. I think that could also be helpful in including more legend/instruction type information with the plots without making the pages too long. 

Another thought is that the feature/landmark table could be made much shorter by only including RMs that contain features. Or even made into a static table - and then include that beneath the dry segs plot. It'd be short enough that it wouldn't need multiple pages, and anyway the principal use case I see for that table is to cross-reference with the plot.

**Comment by rhull21** [comment]
@agropunk remove the tables from the plot templates
@rhull21 will remove from context
@rhull21 will figure out caching issue
@rhull21 will load the feature / landmark table as a short static table

**Comment by rhull21** [comment]
@rhull21 will provide a short table of gages and gage locations under the discharge figure

**Comment by rhull21** [comment]
@agropunk I did some testing over the weekend and it turns out the issue is not related to the database, data cacheing, or (even) figure cacheing. The problem is that plotly's [offline library, `plot`,](https://community.plotly.com/t/offline-plotting-in-python-is-very-slow-on-big-data-sets/3077) is just really slow for large datasets, especially for heatmap (we've got 200,000+ cells in ours, if you think across date, RM, and year). I'll need to do some thinking about how to handle this. The answer will come from either: 
1. passing less information into the figure, to decrease cost
2. thinking about a way to improve performance through multi-threading processing or something. 

Anyway, just wanted this update to be known. It's still an issue. Good thinking, though, taking the tables out of these pages.
